docs_zh-CN/tutorials/finetune.md

```diff
@@ -22,71 +31,166 @@ _base_ = [
 ]
 ```
 
-除此之外，用户也可以直接编写完整的配置文件，而不是使用继承，例如 `configs/mnist/lenet5.py`。
+除此之外，你也可以不使用继承，直接编写完整的配置文件，例如 `configs/mnist/lenet5.py`。
+
+## 修改模型
 
-## 修改分类头
+在进行模型微调是，我们通常希望在主干网络（backbone）加载预训练模型，再用我们的数据集训练一个新的分类头（head）。
 
-接下来，新的配置文件需要按照新数据集的类别数目来修改分类头的配置。只需要修改分类头中的 `num_classes` 设置，除了最终分类头之外的绝大部分预训练模型权重都会被重用。
+为了在主干网络加载预训练模型，我们需要修改主干网络的初始化设置，使用
+`Pretrained` 类型的初始化函数。另外，在初始化设置中，我们使用
+`prefix='backbone'` 来告诉初始化函数移除权重文件中键值名称的前缀，比如把
+`backbone.conv1` 变成 `conv1`。方便起见，我们这里使用一个在线的权重文件链接，它
+会在训练前自动下载对应的文件，你也可以提前下载这个模型，然后使用本地路径。
+
+接下来，新的配置文件需要按照新数据集的类别数目来修改分类头的配置。只需要修改分
+类头中的 `num_classes` 设置即可。
 
 ```python
-_base_ = ['./resnet50.py']
 model = dict(
-    pretrained=None,
-    head=dict(
-        type='LinearClsHead',
-        num_classes=10,
-        in_channels=2048,
-        loss=dict(type='CrossEntropyLoss', loss_weight=1.0),
-    ))
+    backbone=dict(
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint='https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_8xb32_in1k_20210831-ea4938fc.pth',
+            prefix='backbone',
+        )),
+    head=dict(num_classes=10),
+)
+```
+
+```{note}
+这里我们只需要设定我们想要修改的部分配置，其他配置将会自动从我们的父配置文件中获取。
 ```
 
 ## 修改数据集
 
-用户可能还需要准备数据集并编写有关数据集的配置。我们目前支持 MNIST，CIFAR 和 ImageNet 数据集。为了在 CIFAR10 数据集上进行微调，考虑到其原始输入大小为 32，而在 ImageNet 上预训练模型的输入大小为 224，因此我们应将其大小调整为 224。
+当针对一个新的数据集进行微调时，我们通常都需要修改一些数据集相关的配置。比如这
+里，我们就需要把 CIFAR-10 数据集中的图像大小从 32 缩放到 224 来配合 ImageNet 上
+预训练模型的输入。这一需要可以通过修改数据集的预处理流水线（pipeline）来实现。
```

Comment:
a link to [build owner pipeline](https://mmclassification--466.org.readthedocs.build/en/466/tutorials/data_pipeline.html) could be add here.

Reply:
I think "custom data pipeline" is not relative to fine-tune. It's better to be mentioned at a start tutorial.